scripts/lint_jjb.py

```diff
@@ -1,18 +1,21 @@
 #!/usr/bin/env python
 
-# Script used to review jenkins jobs and report job names
-# not following conventions
-# https://github.com/rcbops/rpc-gating#naming-conventions
+# Script used to review jenkins jobs for compliance with internal conventions:
+#   * Naming convetions:
+#        https://github.com/rcbops/rpc-gating#naming-conventions
+#   * Retention Policy
 
 import argparse
 import os
 import re
 import sys
 import yaml
 
+import jmespath
```

Comment:
Do we need to specify this in requirements/constraints?

Reply:
Already in both, as used in other scripts.